lib/ecto/migration.ex

```diff
@@ -246,6 +246,8 @@ defmodule Ecto.Migration do
   ## Options
 
     * `:primary_key` - when false, does not generate primary key on table creation
+    * `:storage_engine` - set the storage engine for MySQL adapters. When not
```

Comment:
We can just call it engine. In the documentation, we should just say: "It customizes the table storage for supported daabases. For MySQL, the default is InnoDB". The idea is that the general documentation comes first, notes for specific databases is added later.